pkg/orchestration/wiring/ec2compute/v2/plugin.go

```diff
@@ -159,7 +194,27 @@ func constructComputeParameters(origSpec *runtime.RawExtension, iamRoleRef, iamI
 	return rawExtension, false, false, nil
 }
 
-func WireUp(stateResource *orch_v1.StateResource, context *wiringplugin.WiringContext) wiringplugin.WiringResult {
+func New() *WiringPlugin {
```

Comment:
The members of this struct and it's use are the only changes to this class, everything else is just moving code from the extra package.